cassandra-scheduler/src/test/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskOfferRequirementProviderTest.java

```diff
@@ -239,10 +239,10 @@ public void testGetNewOfferRequirement() throws Exception {
             config.getExecutorConfig().getExecutorLocation().toString(),
             cmd.getUrisList().get(0).getValue());
         Assert.assertEquals(
-            config.getExecutorConfig().getCassandraLocation().toString(),
+            config.getExecutorConfig().getLibmesosLocation().toString(),
```

Comment:
Can we sort this list in the test so that we don't have to change this ordering every time?